content/Building Engineering Goals.md

```diff
@@ -0,0 +1,66 @@
+
+
+## Improvement opportunities
+
+<img src="images/team_engagement.jpg" alt="Team engagement" width="500"/>
+
+### Increase team engagement
+Being transparent with the team about department goals is a must. A good presentation with that info works for that purpose. We encourage our team to raise their hands if they have questions, and we create opportunities to promote their participation.  
+But keeping the team involved with company goals is much more difficult. We want them to be engaged. We also want to grow their feeling of ownership for their work and their contribution to the company.
+
+**Our hypothesis.** Building goals after a couple of leader meetings and then giving a presentation to the whole team is a top-down approach. Creating a presentation without the team’s input makes them view our engineering goals as just company goals and not their own. This approach doesn’t generate a feeling of ownership.
+
+**A possible solution.** We need to engage the team in the process of developing company goals. Let their knowledge and experience contribute to deciding the best approach to any engineering problems that might arise.
+
+### Take advantage of our main asset
+As mentioned, our team is larger and more skilled than ever before. It’s crucial to build the right engineering goals and thought strategies to face potential challenges. Even with great managers and leaders in our company, not involving our team would be a big mistake.
+
+**A possible solution.** It might seem obvious, but we need to take advantage of our most valuable resources, our people. We need to involve the team in talks to define our plan for overcoming hurdles and making every project a success.
+
+
+## A new approach
+A new year was coming, and it was time to create new goals. We were anxious to move to the next level by taking advantage of our lessons learned in the last year. But it wasn’t only a matter of involving the team. We also needed a good framework to effectively capitalize on their ideas and make them valuable action items.
+
+We learned two important lessons: 
+* Increase team engagement with goals. 
+* Take advantage of the company’s main asset, the team.
+
+## World Cafe
+A world cafe is a collaborative meeting place for a large group dialog. The whole team discusses strategic topics and possible solutions. In our case, it was a three-hour event. The entire engineering team participated, about 70 people. This meeting wasn’t mandatory, but it was a good sign that everyone was there. This investment might sound big in terms of hours and budget. But after some time passed, we saw that the investment was completely worth it.
+
+<img src="images/world_cafe.png" alt="Team engagement" width="400"/>
```

Comment:
It is ok to have the same alt text for this picture?

Reply:
I don't think so, good catch, thanks